nuget/spec/fixtures/packages_configs/packages.config

```diff
@@ -8,6 +8,6 @@
   <package id="NuGet.Core" version="2.11.1" targetFramework="net46" />
   <package id="NuGet.Server" version="2.11.2" targetFramework="net46" />
   <package id="RouteMagic" version="1.3" targetFramework="net46" />
-  <package id="WebActivatorEx" version="2.1.0" targetFramework="net46" />
+  <package id="WebActivatorEx" version="2.1.0" targetFramework="net46"></package>
```

Comment:
I assume this test fails w/o this patch?

Reply:
Correct

Reply:
I got the tests running locally and was able to confirm :)